examples/autocomplete-states.example.json

```diff
@@ -1,9 +1,63 @@
 {
```

Comment:
nit: let's revert the formatting to what it used to be? probably accidental formatting?